phys2bids/tests/test_bids_units.py

```diff
@@ -0,0 +1,12 @@
+from phys2bids.bids_units import bidsify_units
+
+
+def test_bidsify_units():
+    # test normal unit
+    assert bidsify_units("V") == "v"
```

Comment:
```suggestion
    assert bidsify_units('V') == 'v'
```

Same as before. I know it's _peccata minuta_, but since the rest of phy2bids uses `'` for strings, it would be nice to maintain the same style.
Same for the following lines.